ambry-router/src/test/java/com/github/ambry/router/CompressionMetricsTest.java

```diff
@@ -0,0 +1,19 @@
+package com.github.ambry.router;
```

Comment:
license

Reply:
Yes :)  I missed this file too.  I'll add it.